client/jetpack-cloud/sections/onboarding-tours/dashboard-walkthrough-tour/index.tsx

```diff
@@ -9,6 +10,35 @@ export default function DashboardWalkthroughTour() {
 
 	const urlParams = new URLSearchParams( window.location.search );
 	const shouldRenderDashboardTour = urlParams.get( 'tour' ) === 'dashboard-walkthrough';
+	// TODO: As soon as v1 is no longer in use we should
+	// Define targets for both versions of the dashboard
+	const dashboardTargets = {
```

Comment:
Nesting versioned data is an easy way for data to get messy; I'd instead recommend defining two vars and setting the value of `dashboardTargets` based on the version we'll be using; that would allow us to easily chop the old values without having to adjust the nesting levels (or worse yet, keeping them as a legacy artifact).